packages/loopring_v3/contracts/impl/libexchange/ExchangeAccounts.sol

```diff
@@ -140,6 +140,18 @@ library ExchangeAccounts
         );
     }
 
+    function calculateAccountId(uint24 accountIdx)
```

Comment:
@Brecht, do you think it's a good idea to introduce a concept of TreeSharding

interface TreeSharding
{
     function accountIDFromAccountIndex(uint24 accountIndex) returns (uint24 accountID);
}